stock_picking_purchase_order_alert/models/stock_picking.py

```diff
@@ -0,0 +1,146 @@
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+logger = logging.getLogger(__name__)
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    has_quantity_alert = fields.Boolean(
+        string="Quantity Alert",
+        compute="_compute_has_quantity_alert",
+        store=True,
+        help="Indicates if any move line exceeds ordered "
+        "quantity by more than threshold",
+    )
+    quantity_alert_message = fields.Html(
+        string="Quantity Alert Message",
+        compute="_compute_has_quantity_alert",
+        store=True,
+    )
+
+    @api.depends(
+        "move_ids_without_package",
+        "move_ids_without_package.quantity_done",
```

Comment:
I'm having the next situation:

1. Try to receipt over the percentage
2. Get the error
3. Change the qty at purchase line level to match the same value
4. Try to receipt
5. Still getting the error
```suggestion
        "move_ids_without_package.quantity_done",
        "move_ids_without_package.product_uom_qty",
```